index.js

```diff
@@ -70,6 +73,29 @@ async function run() {
       })
     }
 
+    if (logConfigurationLogDriver) {
+      if (!containerDef.logConfiguration) { containerDef.logConfiguration = {} }
+      const validDrivers = ["json-file", "syslog", "journald", "gelf", "fluentd", "awslogs", "splunk", "awsfirelens"];
```

Comment:
Currently for the ECS task, these are the log configurations that are supported.

For tasks on AWS Fargate, the supported log drivers are `awslogs, splunk, and awsfirelens`
For tasks hosted on Amazon EC2 instances, the supported log drivers are `awslogs, fluentd, gelf, json-file, journald, logentries, syslog, splunk, and awsfirelens`.

Are we assuming these drivers for ECS Tasks based on Ec2 instance? if this is the case do we need to add `logentries` as well ?

Reply:
my current scenario is for fargate but yes, let's add logentries!